pkg/config/leaderelection/leaderelection.go

```diff
@@ -124,3 +124,13 @@ func LeaderElectionDefaulting(config configv1.LeaderElection, defaultNamespace,
 	}
 	return ret
 }
+
+// LeaderElectionSNOConfig uses the formula derived in LeaderElectionDefaulting with increased
+// retry period and lease duration for SNO clusters that have limited resources
+func LeaderElectionSNOConfig(config configv1.LeaderElection) configv1.LeaderElection {
```

Comment:
Can you run the math on these numbers to produce 
1. clock skew tolerance
2. kube-apiserver downtime tolerance
3. worst non-graceful lease acquisition
4. worst graceful lease acquisition

And add a comment to the code please.

Reply:
Absolutely, added.